ssh-server/main.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net"
+	"os"
+	"os/exec"
+
+	"github.com/kr/pty"
+	"golang.org/x/crypto/ssh"
+)
+
+var (
+	serverPort = flag.Uint("port", uint(22), "the port to listen on")
+	password   = flag.String("password", "", "the client authentication password")
+)
+
+func main() {
+	flag.Parse()
+
+	if envpwd := os.Getenv("password"); *password == "" && envpwd != "" {
```

Comment:
to make it easier to run it from the command line? I started to implement it with command line and I switched to env variable because I didn't want the password in the log history. I'll remove the flag option